contrib/security/PSS/static/baseline/kustomization.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: kustomize.config.k8s.io/v1alpha1
+kind: Component
+
+resources:
+- ../../../../../common/kubeflow-namespace/base
```

Comment:
The two resources here are already in example.yaml. Why do you list them here as well?

You should add this component here commented out to example.yaml instead as planned in the Google document..